x/interchaintxs/client/cli/tx.go

```diff
@@ -67,10 +67,10 @@ func RegisterInterchainAccountCmd() *cobra.Command {
 
 func SubmitTxCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "submit-tx [connection-id] [owner] [path/to/sdk_msgs.json]",
+		Use:     "submit-tx [connection-id] [owner] [path/to/sdk_msgs.json] [memo]",
```

Comment:
Hmmm, this help message implies (people usually use square brackets to mark optional arguments) that connection-id, owner, path and memo are all optional arguments, but they are not. I am providing my own suggestion but I am not really sure about it neither.

```suggestion
		Use:     "submit-tx connection-id owner path/to/sdk_msgs.json [memo]",
```

Reply:
It's a common practice in Cosmos-SDK to wrap arguments in square brackets. So I'd stick with current version

Reply:
But I'd personally pass memo via flag, not argument. This shows that memo is optional

Reply:
Well, if it is a common practice, then I am leaving an approving review. And at the same time anyone is free to change arguments hint if desirable.

Reply:
Changed to flag, did I do it ok?)

Reply:
looks alright, though @pr0n00gler told us that:
> It's a common practice in Cosmos-SDK to wrap arguments in square brackets
so I am not 100% sure.

@pr0n00gler what do you think about it?

Reply:
I think we should return square brackets here for consistency